tensorflow/go/.gitignore

```diff
@@ -1,2 +1,2 @@
 # .pb.go files generated by generate.sh
-internal/proto/*
+pb/proto/*
```

Comment:
IIUC, the `pb` directory is one level up, so should the `.gitignore` of the parent dir be changed instead?
Though see comment later on.